homeassistant/components/image_processing/dlib_face_detect.py

```diff
@@ -0,0 +1,69 @@
+"""
+Component that will help set the dlib face detect processing.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/image_processing.dlib_face_detect/
+"""
+import logging
+import io
+
+from homeassistant.core import split_entity_id
+from homeassistant.components.image_processing import (  # noqa
+    PLATFORM_SCHEMA, CONF_SOURCE, CONF_ENTITY_ID, CONF_NAME)
+from homeassistant.components.image_processing.microsoft_face_identify import (
+    ImageProcessingFaceEntity)
```

Comment:
Should we move this entity to the main component? Also, should it be faces or can it be generalized to be just "Things" ?